examples/InteractiveExample.tsx

```diff
@@ -187,20 +177,13 @@ export default class InteractiveExample extends React.Component<Props, State> {
         } else {
           this.highlight({
             type: HighlightType.LOCATION,
-            locationId: getLocationId(object),
+            locationId: this.props.getLocationId!(object),
```

Comment:
The usage of `!` can result to a crash since there are no defaultProps specified for this component.